test/equiv_yosys.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
I think it makes sense to put this under circt/utils.  Please add the standard file header boilerplate.  I also think it would make sense to name this `equiv-yosis.sh` (LLVM prefers hyphens instead of underscores in tool names) or perhaps just `equiv-check` since yosis is an implementation detail of the script.